src/main/resources/templates/login.html

```diff
@@ -1,68 +1,68 @@
-                <label class="form-label" for="usernameOrEmail">Username or email address</label>
-            </div>
-
-            <!-- Password input -->
-            <div class="form-outline mb-4">
-                <input type="password"
-                       id="password"
-                       th:name="password"
-                       class="form-control"/>
-                <label class="form-label" for="password">Password</label>
-            </div>
+        <hr>
+        <!-- Email input -->
+        <div class="form-floating ">
+            <input type="text"
+                   id="usernameOrEmail"
+                   th:name="usernameOrEmail"
+                   placeholder="Username or email address"
+                   class="form-control"/>
+            <label class="form-label" for="usernameOrEmail">Username or email address</label>
+        </div>
 
-            <!-- 2 column grid layout for inline styling -->
-            <div class="row mb-4">
-                <div class="col d-flex justify-content-center">
-                    <!-- Checkbox -->
-                    <div class="form-check">
-                        <input class="form-check-input" type="checkbox" value="" id="form2Example31" checked/>
-                        <label class="form-check-label" for="form2Example31"> Remember me </label>
-                    </div>
-                </div>
+        <!-- Password input -->
+        <div class="form-floating ">
+            <input type="password"
+                   id="password"
+                   th:name="password"
+                   placeholder="Password"
+                   class="form-control"/>
+            <label class="form-label" for="password">Password</label>
+        </div>
 
-                <div class="col">
-                    <!-- Simple link -->
-                    <a href="#">Forgot password?</a>
+        <!-- 2 column grid layout for inline styling -->
+        <div class="row mt-3">
+            <div class="col d-flex justify-content-center">
+                <!-- Checkbox -->
+                <div class="form-check">
+                    <input class="form-check-input" type="checkbox" value="" id="form2Example31" checked/>
+                    <label class="form-check-label" for="form2Example31"> Remember me </label>
                 </div>
             </div>
 
-            <div class="btn-toolbar">
-                <!-- Submit button -->
-                <div class="btn-group">
-                    <button type="submit" class="btn btn-primary btn-block mb-4">Sign in</button>
-                    <a th:href="@{/auth/register}" class="btn btn-success btn-block mb-4">New account</a>
-                </div>
+            <div class="col">
+                <!-- Simple link -->
+                <a href="#" class="link-warning text-decoration-none">Forgot password?</a>
             </div>
+        </div>
+
+        <div class="d-flex align-items-center justify-content-start mt-5">
```

Comment:
I think it would be better to use justify-content-between

Reply:
If the second element was a button too, it would look good, but the second element is a link.
We can define another button in the future: "Reset" and realign the content 
![login form](https://user-images.githubusercontent.com/44172060/235291843-eac848e5-8fd6-410e-a291-3f00c52af7bf.png)